get_soccer_players_articles.py

```diff
@@ -11,20 +11,44 @@
 
 
 def load_wiki_ids(filein):
+    """Load soccer player articles Wiki IDs from a file.
+   
+    Keyword arguments:
```

Comment:
This is not a *keyword* argument, this is a function *parameter*.
You should write docstrings according to [PEP 287](https://www.python.org/dev/peps/pep-0287/).
If you use an IDE like **PyCharm**, this should be pretty easy, see https://www.jetbrains.com/pycharm/help/creating-documentation-comments.html